c_glib/doc/gandiva-glib/gandiva-glib-docs.xml

```diff
@@ -48,6 +48,14 @@
     </chapter>
   </part>
 
+  <part id="tree-expression-builder">
+    <title>Tree Expression Builder</title>
```

Comment:
How about "Expression Tree"? We hide tree expression builder in implementation. So it's better that we also hide it in document.

Reply:
That makes sense. I've changed it.